test/test_tensordict.py

```diff
@@ -1465,6 +1465,20 @@ def test_repr(self, td_name, device):
         td = getattr(self, td_name)(device)
         _ = str(td)
 
+    def test_set_default_missing_key(self, td_name, device):
+        torch.manual_seed(1)
```

Comment:
I think it's all deterministic in your test no?

Reply:
Yes, true! Forgot to remove it after copy-pasting another test.